iOS/Core/FeatureFlag.swift

```diff
@@ -155,7 +155,8 @@ extension FeatureFlag: FeatureFlagDescribing {
              .canScanUrlBasedSyncSetupBarcodes,
              .canInterceptSyncSetupUrls,
              .removeWWWInCanonicalizationInThreatProtection,
-             .june2025TabManagerLayoutChanges:
+             .june2025TabManagerLayoutChanges,
+             .setAsDefaultBrowserPiPVideoTutorial:
```

Comment:
btw, if you set this to default to false just now, you can go ahead and merge to main any time you like.  And then in the ship review just point people at the latest alpha build. 

once approved, quick PR to switch from default false, to default true.

cc @graeme